code/game/objects/items/shields.dm

```diff
@@ -159,20 +159,24 @@
 	armor = list("melee" = 25, "bullet" = 25, "laser" = 5, "energy" = 0, "bomb" = 30, "bio" = 0, "rad" = 0, "fire" = 70, "acid" = 80)
 	lefthand_file = 'icons/mob/inhands/equipment/shields_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/equipment/shields_righthand.dmi'
-	item_state = "makeshift_shield"
+	icon = 'icons/obj/items_and_weapons.dmi'
+	item_state = "metal"
+	icon_state = "makeshift_shield"
 	materials = list(MAT_METAL = 18000)
 	slot_flags = null
-	block_chance = 25
-	force = 5
+	block_chance = 35

```

Comment:
this shouldn't be superior to the riot shield

Reply:
?
Riot are 50%
And wooden ones are 30%
This one takes more to make then wooden shields